src/api/bank-accounts/bank-account-approval-activity.md

```diff
@@ -0,0 +1,93 @@
+---
+layout:  default
+grand_parent: API Reference
+parent: Bank Accounts
+title: Bank Account Approval Activity
+permalink: /api/bank-account-approval-activity
+---
+
+# Bank Account Approval Activity
+{:.no_toc}
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Bank Account Approval Activity
+{% h4 Mandatory Fields %}
+
+|     Field      |        Type        |                             Description                              |
+| :------------- | :----------------- | :------------------------------------------------------------------- |
+| activityNumber | Number             | Unique sequential Bank Account Approval Activity number.             |
+| approvalId     | String             | The id of the associated [Bank Account Approval][].                  |
+| activityType   | String             | The type of the Bank Account Approval Activity.                      |
+| createdAt      | {% dt Timestamp %} | When the Bank Account Approval Activity was created.                 |
+| createdBy      | {% dt CRN %}       | The User or API Key that created the Bank Account Approval Activity. |
+
```

Comment:
@GeekOnGadgets @ncjones I worked with Andy on this and we documented the Bank Account Approval Activity as this is what the endpoints are related to in Notion. I'm not sure if they should just be documented under the Bank Account Approval

Reply:
Yes, this is part of bank account approval.